src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private String type;
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
```

Comment:
Consider adding validation in the constructor to ensure that 'quantity' is non-negative, as negative quantities may not make sense in the context of a transaction.